client/homescreen/utils.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * External dependencies
+ */
+import { OPTIONS_STORE_NAME } from '@woocommerce/data';
+
+export function taskListsHidden( select ) {
```

Comment:
Hmm, I'm not sure there is value in separating this out, its only called once? The functionality also feels like it doesn't belong in a "utils" type file. This is a nitpick though, I just don't like adding lots of extra files and moving logic around esp if its only called one time.

Reply:
I like the idea I removed the utils file in the commit 1ed7780